packages/now-build-utils/src/detect-routes.ts

```diff
@@ -289,18 +306,23 @@ function getPublicBuilder(builders: Builder[]): Builder | null {
 
 export async function detectRoutes(
   files: string[],
-  builders: Builder[]
+  builders: Builder[],
+  featHandleMiss: boolean
 ): Promise<RoutesResult> {
-  const routesResult = await detectApiRoutes(files, builders);
+  const routesResult = await detectApiRoutes(files, builders, featHandleMiss);
   const publicBuilder = getPublicBuilder(builders);
 
   if (routesResult.defaultRoutes && publicBuilder) {
     const directory = publicBuilder.src.replace('/**/*', '');
 
-    routesResult.defaultRoutes.push({
+    const route: Source = {
       src: '/(.*)',
       dest: `/${directory}/$1`,
-    });
+    };
+    if (featHandleMiss) {
+      route.check = true;
+    }
+    routesResult.defaultRoutes.push(route);
```

Comment:
To confirm, this comes after handle: miss right?

Reply:
Indeed, but I updated the code to make it more clear